src/common/segment-utils.ts

```diff
@@ -104,3 +105,63 @@ type FacebookBasicEvent =
 export const trackFacebookBasicEvent = (eventName: FacebookBasicEvent) => {
   analytics.track(eventName, {}, { integrations: { All: false, 'Facebook Pixel': true } });
 };
+
+/**
+ * Register a Segment plugin that will parse url paths from learn.xip.co.
+ *
+ * @see https://github.com/segmentio/analytics-next/tree/master/packages/browser#-plugins
+ *
+ */
+export const urlPathFilter: Plugin = {
+  name: 'Clickfunnels Path Filter',
+  type: 'enrichment',
```

Comment:
What does the type: 'enrichment' mean here?

Reply:
https://github.com/segmentio/analytics-next/tree/master/packages/browser#-plugins
It's mentioned here as a type for the plugin in segment. It's used in the case that you want to 'enrich' event data using a plugin.